torchci/clickhouse_queries/queued_jobs_by_label/query.sql

```diff
@@ -1,6 +1,4 @@
 --- This query is used by HUD metrics page to get the list of queued jobs grouped by their labels
-SET
```

Comment:
Q: had something like that be detected by linter of sorts?

Reply:
It's valid, but needs to enable a setting for multiple statements we probably don't want to enable.
It works in CH UI.